src/main/java/de/qabel/desktop/ui/util/Icons.java

```diff
@@ -0,0 +1,31 @@
+package de.qabel.desktop.ui.util;
+
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+
+public class Icons {
```

Comment:
Looks fine to me, I don't care much if the icon paths live in some xml/properties-file or directly as statics in the class.